common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -154,90 +154,107 @@ div.problem {
   }
 }
 
-// +Problem - Choice Group
-// ====================
-div.problem {
-  .choicegroup {
-    @include clearfix();
-    min-width: 100px;
-    width: auto !important;
-    width: 100px;
-
-    label {
-      @include box-sizing(border-box);
-      display: inline-block;
-      clear: both;
-      margin-bottom: ($baseline/2);
-      border: 2px solid $gray-l4;
-      border-radius: 3px;
-      padding: ($baseline/2);
-      width: 100%;
-
-      &.choicegroup_correct {
-        @include status-icon($correct, "\f00c");
-        border: 2px solid $correct;
+// Choice Group - silent class
+%choicegroup-base {
+  @include clearfix();
+  min-width: 100px;
+  width: auto !important;
+  width: 100px;
+
+  label {
+    @include box-sizing(border-box);
+    display: inline-block;
+    clear: both;
+    margin-bottom: ($baseline/2);
+    border: 2px solid $gray-l4;
+    border-radius: 3px;
+    padding: ($baseline/2);
+    width: 100%;
 
-        &:after {
-          @include margin-left($baseline*.75);
-        }
+    &.choicegroup_correct {
+      @include status-icon($correct, "\f00c");
+      border: 2px solid $correct;
 
-        // keep green for correct answers on hover.
-        &:hover {
-          border-color: $correct;
-        }
+      &:after {
+        @include margin-left($baseline*.75);
       }
 
-      &.choicegroup_partially-correct {
-        @include status-icon($partiallycorrect, "\f069");
-        border: 2px solid $partiallycorrect;
+      // keep green for correct answers on hover.
+      &:hover {
+        border-color: $correct;
+      }
+    }
 
-        &:after {
-          @include margin-left($baseline*.75);
-        }
+    &.choicegroup_partially-correct {
+      @include status-icon($partiallycorrect, "\f069");
+      border: 2px solid $partiallycorrect;
 
-        // keep green for correct answers on hover.
-        &:hover {
-          border-color: $partiallycorrect;
-        }
+      &:after {
+        @include margin-left($baseline*.75);
       }
 
-      &.choicegroup_incorrect {
-        @include status-icon($incorrect, "\f00d");
-        border: 2px solid $incorrect;
+      // keep green for correct answers on hover.
+      &:hover {
+        border-color: $partiallycorrect;
+      }
+    }
 
-        &:after {
-          @include margin-left($baseline*.75);
-        }
+    &.choicegroup_incorrect {
+      @include status-icon($incorrect, "\f00d");
+      border: 2px solid $incorrect;
 
-        // keep red for incorrect answers on hover.
-        &:hover {
-          border-color: $incorrect;
-        }
+      &:after {
```

Comment:
nitpick: although this is legacy code, you can move this 

&:after {
    @include margin-left($baseline*.75);
}

out of each: correct, incorrect, and partially-correct, and put it above : hover, because it applies to the label.

👍  once this is cleaned up.

good work.